package.json

```diff
@@ -13,12 +13,13 @@
     "@google/local-home-sdk": "^1.0.0",
     "@typescript-eslint/eslint-plugin": "^3.3.0",
     "@typescript-eslint/parser": "^3.3.0",
-    "ts-node": "^8.10.2",
     "ava": "^2.1.0",
+    "cbor": "^5.0.2",
     "eslint": "^7.2.0",
     "eslint-config-google": "^0.14.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-prettier": "^3.1.4",
+    "ts-node": "^8.10.2",
```

Comment:
newer version of ava shouldn't need this, see:
https://github.com/avajs/typescript

Reply:
Will remove

Reply:
Will address in #30